test/Microsoft.Azure.WebJobs.Extensions.Sql.Tests.csproj

```diff
@@ -61,5 +61,16 @@
     </ItemGroup>
     <Copy SourceFiles="@(_PythonCopyItems)" DestinationFolder="$(OutDir)\SqlExtensionSamples\Python\%(RecursiveDir)" />
     <Message Text="Copied Python Samples output to $(OutDir)\SqlExtensionSamples\Python" Importance="high" />
+    <ItemGroup>
+      <_CSharpScriptCopyItems Include="..\samples\samples-csharpscript\InputBindingSamples\**\*.*" />
```

Comment:
Can we copy the whole samples-csharpscript folder instead of the specific folders here?

Reply:
I should have followed the JavaScript structure of samples (without dividing the samples into InputBindings and OutputBindings like CSharp) But for now I can't do a simple copy of the entire folder unless I add custom logic to calculate the workding directory for CShaprScript test runs.